src/env.js

```diff
@@ -205,6 +208,7 @@ Sk.configure = function (options) {
     Sk.switch_version("round$", Sk.__future__.dunder_round);
     Sk.switch_version("next$", Sk.__future__.dunder_next);
     Sk.switch_version("clear$", Sk.__future__.list_clear);
+    Sk.switch_version("copy$", Sk.__future__.list_copy);
```

Comment:
And do the same here:

```suggestion
    Sk.switch_version("copy$", Sk.__future__.python3);
```